src/app/app.module.ts

```diff
@@ -88,10 +96,23 @@ import { PillComponent } from './pill/pill.component';
     MaterialModule,
     FlexLayoutModule,
     HttpClientModule,
+    AngularFireModule.initializeApp(firebaseConfig.firebase, 'firebaseApp'),
```

Comment:
I used it for debugging when I had problems with the initialization and then forgot to change it back - changed it now since it doesn't have any benefit.